config/events/watchers/slack_if_cron_heartbeat_isnt_running.rb

```diff
@@ -0,0 +1,9 @@
+Houston.config.on "watcher:success" => "watcher:notify-if-cron_inactive" do
```

Comment:
I like this as a gateway to more insight and a "dashboard" type information. We could probably evolve this to have response time, etc in the future. ⭐️